wdl-engine/src/backend.rs

```diff
@@ -93,44 +96,50 @@ pub struct TaskExecutionRoot {
 
 impl TaskExecutionRoot {
     /// Creates a task execution root for the given path and execution attempt.
-    pub fn new(path: &Path, attempt: u64) -> Result<Self> {
-        let path = absolute(path).with_context(|| {
+    pub fn new(path2: &Path, attempt: u64) -> Result<Self> {
```

Comment:
Maybe I'm missing something obvious. Why is this `path2`? What's the `2` mean?

Reply:
Whoops, that was a rename to see where it was being used for corrections, it should be `path`.